Runtime/SimvaPlugin.cs

```diff
@@ -115,36 +124,57 @@ public bool WantsToQuit()
 
         public void Demo()
         {
-            SceneManager.LoadSceneAsync(GamePlayScene);
+            Log("Starting Demo Gameplay");
+            StartGameplay();
         }
 
         public void OnAuthUpdated(OAuth2Token token)
         {
             lastAuth = token;
         }
 
-        public void RunScene(string name)
+        public void RunScene(string sceneName)
         {
-            if (SceneManager.GetActiveScene().name != SimvaScene)
+            var name="";
+            switch (sceneName)
             {
-                SceneManager.LoadSceneAsync(SimvaScene).completed += ev =>
```

Comment:
If there is no "SimvaScene" loaded, it might work on startup, but should fail when after gameplay is launched if the user does not trigger the scene. What's the reasoning on removing loading the "SimvaScene"?

Reply:
The trigger is done now by calling RunScene(Simva.***) in the code of the plugin to run the simva scene and as that we don't need anymore "SceneManager.LoadSceneAsync(SimvaScene).completed += ev =>"

Reply:
Although I've checked this does not impact uAdventure, I'm concerned about the SimvaScene because it is a scene that is mostly empty and its used as cleanup. This allows the SimvaPlugin to switch from gameplay to Simva environment when needed having no other game objects in the scene (except those DontDestroyOnLoad). This can avoid collisions with menus, background music, etc.
I would suggest finishing the review and requesting a new feature to allow for loading the simva scenes on a blank scene or additively (as it is right now).